app/src/main/java/com/poti/android/presentation/history/component/HistoryParticipantOverview.kt

```
@@ -0,0 +1,101 @@
+                sizeType = PotiItemOptionSize.SMALL,
+                text = deliveryMethod,
+            )
+            PotiItemOption(
+                optionType = PotiItemOptionType.PRICE,
+                sizeType = PotiItemOptionSize.SMALL,
+                text = priceText(price),
+            )        }
+    }}
+
+// TODO: [천민재] 임시 util 함수
+private fun priceText(price: Int) = String.format("%,d원", price)
```

detekt (1.23.8)

[warning] String.format("%,d원", price) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)